samples/application-insights/sample.md

```diff
@@ -18,7 +18,9 @@ For a detailed explanation of these metrics refer to the [metrics captured secti
 
 ## Prerequisites
 
-An Application Insights account and instrumentation key. The instrumentation key can be obtained under the Application Insights instance properties pane in the Azure portal.
+To run this sample, an Application Insights account and instrumentation key are required.
+
+Note: for simplicity, the sample uses the Learning transport. 
```

Comment:
Although the sample uses Azure Application Insights, the solution itself does not have to run on an Azure message transport. This example uses the [Learning Transport](/transports/learning/) but could be modified to run on any [transport](/transports/).